### ISSUE TITLE
Keyboard control of progress bar not working on iPad with Voice Over

### ISSUE THREAD

**Issue by vdeshpande** [videojs/video.js#3076]
We need to be able to find a way to control the progress bar via the keyboard, when running VoiceOver on iPad, when using an external keyboard.

**Comment by gkatsev** [comment]
I believe that it works as best it can be using our custom element. Double tap or long press should be able to seek but using stuff like the volume buttons won't work because it isn't a native slider element, unfortunately.
